programs/bubblegum/program/src/lib.rs

```diff
@@ -251,6 +251,64 @@ pub struct Delegate<'info> {
     pub system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+pub struct UpdateMetadata<'info> {
+    /// CHECK: Can optionally specify the old_metadata of the leaf through an account to save transaction space
+    /// CHECK: This account is checked in the instruction
+    pub old_metadata_acct: Option<UncheckedAccount<'info>>,
```

Comment:
We could name this `metadata_buffer` to be a bit more informative.

Reply:
done